src/utils/simplified-account/navigation/service-settings-navigation.ts

```diff
@@ -0,0 +1,188 @@
+import paths from '@root/paths'
+import formatServiceAndAccountPathsFor from '@utils/simplified-account/format/format-service-and-account-paths-for'
+import { NavigationBuilder, NavigationCategories } from '@utils/simplified-account/navigation/NavigationBuilder.class'
+import { WORLDPAY, STRIPE } from '@models/constants/payment-providers'
+import GatewayAccount from '@models/GatewayAccount.class'
+import Service from '@models/Service.class'
+import { LIVE } from '@models/constants/go-live-stage'
+import UserPermissions from '@models/user/permissions'
+
+export = (account: GatewayAccount, service: Service, currentUrl: string, permissions: Record<string, boolean>) => {
+  const navBuilder = new NavigationBuilder(currentUrl, permissions)
+  const serviceSettings = navBuilder
+    .category('about your service', { collapsible: true })
+    .add({
+      id: 'service-name',
+      name: 'service name',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.serviceName.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: UserPermissions.settings.serviceName.serviceNameUpdate,
+    })
+    .add({
+      id: 'email-notifications',
+      name: 'email notifications',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.emailNotifications.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: UserPermissions.any,
+      alwaysViewable: true,
+    })
+    .add({
+      id: 'team-members',
+      name: 'team members',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.teamMembers.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: UserPermissions.any,
+    })
+    .add({
+      id: 'organisation-details',
+      name: 'organisation details',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.organisationDetails.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: UserPermissions.settings.merchantDetails.merchantDetailsUpdate,
+    })
+    .category('payment provider', { collapsible: true })
+    .add({
+      id: 'stripe-details',
+      name: 'stripe details',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.stripeDetails.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: UserPermissions.settings.stripe.stripeAccountDetailsUpdate,
+      conditions: account.paymentProvider === 'stripe' && account.type === 'live',
+    })
+    .add({
+      id: 'worldpay-details',
+      name: 'worldpay details',
+      path: formatServiceAndAccountPathsFor(
+        paths.simplifiedAccount.settings.worldpayDetails.index,
+        service.externalId,
+        account.type
+      ),
+      hasPermission: 'gateway_credentials_read',
```

Comment:
can this (and some others below) be replaced with permissions defined in `UserPermissions`?